packages/realm/src/platform/network.ts

```diff
@@ -60,5 +60,5 @@ export const network: NetworkType = {
 };
 
 export function inject(injected: NetworkType) {
-  Object.freeze(Object.assign(network, injected));
+  Object.assign(network, injected);
```

Comment:
Did using `.freeze()` cause an issue here?

Reply:
Unless this `inject` called multiple times, the `freeze` shouldn't be an issue.

Reply:
Yeah - my suggestion to avoid the freeze was only if we choose to expose the `inject` function to end-users.

Reply:
Gotcha 🙂